.setup/SUBMITTY_TEST.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+
+pushd /usr/local/submitty/GIT_CHECKOUT/Submitty/site  > /dev/null || {
+    echo "Failed to change to the Submitty/site directory.
+    Please check if you have /usr/local/submitty/GIT_CHECKOUT/Submitty/site and valid permission"
```

Comment:
What about this instead?
```suggestion
    echo "Failed to change to /usr/local/submitty/GIT_CHECKOUT/Submitty/site."
```